src/Testing.re

```diff
@@ -6,9 +6,19 @@ module Result = {
   type t('value) = {result: current('value)};
 };
 
-[@bs.module "react-hooks-testing-library"]
-external renderHook: ([@bs.uncurry] (unit => 'hook)) => Result.t('hook) =
+module Options = {
+  [@bs.deriving {abstract: light}]
+  type t('props) = {
```

Comment:
There is a type parameter `'props`, but didn't use.

Reply:
Fixed!